poetry.lock

```diff
@@ -427,6 +427,17 @@ files = [
 [package.dependencies]
 pycparser = "*"
 
+[[package]]
+name = "cfgv"
```

Comment:
what is this? @edwinjosegeorge

Reply:
These packages was install as the dependencies of `pre-commit` package under `dev` group. 

![image](https://github.com/BerriAI/litellm/assets/52248484/2b4170a4-41ee-4050-8531-e21c3018d382)